simtools/simtel/simtel_light_emission.py

```diff
@@ -0,0 +1,552 @@
+            z_origin = (
+                self.default_le_config["z_pos_ILLN-01"]["default"]
+                - self.default_le_config["z_pos"]["real"]
+            )
+            _, angles = self.calibration_pointing_direction()
+            angle_theta = angles[0]
+            angle_phi = angles[1]
+            command += f" --laser-position '{x_origin.value},{y_origin.value},{z_origin.value}'"
+
+            command += f" --telescope-theta {angle_theta}"
+            command += f" --telescope-phi {angle_phi}"
+            command += f" --laser-theta {90-angles[2]}"
+            command += f" --laser-phi {angles[3]}"  # convention north (x) towards east (-y)
+
+            # further optional properties not used here:
+            # 'laser_external_trigger', 'laser_pulse_exptime',
+            # 'laser_pulse_offset' 'laser_pulse_twidth'
+
+            command += f" --atmosphere {self.output_directory}/model/"
+            command += f"{self._telescope_model.get_parameter_value('atmospheric_profile')}"
+        command += f" -o {self.output_directory}/{self.le_application[0]}.iact.gz"
+        command += "\n"
+
+        return command
+
+    def _make_simtel_script(self, **kwargs):  # pylint: disable=unused-argument
+        """Return the command to run simtel_array."""
+
+        # LightEmission
+        command = f"{self._simtel_source_path.joinpath('sim_telarray/bin/sim_telarray/')}"
+        command += f" -c {self._telescope_model.get_config_file()}"
+
+        def remove_line_from_config(file_path, line_prefix):
+            with open(file_path, "r", encoding="utf-8") as file:
+                lines = file.readlines()
+
+            with open(file_path, "w", encoding="utf-8") as file:
+                for line in lines:
+                    if not line.startswith(line_prefix):
+                        file.write(line)
+
+        remove_line_from_config(self._telescope_model.get_config_file(), "array_triggers")
+
+        command += " -DNUM_TELESCOPES=1"
+        command += " -I../cfg/CTA"
+        command += "iobuf_maximum=1000000000"
+        command += super()._config_option(
+            "altitude", self._site_model.get_parameter_value("corsika_observation_level")
+        )
+        # command += super()._config_option("maximum_telescopes", "1")
+        # command += super()._config_option("trigger_telescopes", "1")
+        command += super()._config_option(
+            "atmospheric_transmission",
+            self._telescope_model.get_parameter_value("atmospheric_transmission"),
+        )
+        # command += super()._config_option("trigger_current_limit", "1e10")
+        # command += super()._config_option("show", "all") # for debugging
+        # command += super()._config_option("random_state", "none")
+        # command += super()._config_option("ONLY_TRIGGERED_TELESCOPES", "0")
+        # command += super()._config_option("ONLY_TRIGGERED_ARRAYS", "0")
+        command += super()._config_option("TRIGGER_CURRENT_LIMIT", "20")
```

Comment:
Are these (and the following) parameters which should be configured? Or do they never change?

Reply:
Did you see this question?